### PR TITLE
feat: add check-health.yml

### PR DESCRIPTION
**Pull request by brivu**
This PR adds the `check-health.yml` file to the `AWS Health` Orb. It merges into the `alpha` branch. Once the orb is complete and ready for a new release, the `alpha` branch will be merged into main.

### CODE REVIEW

File: src/commands/check-health.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+description: >
+  This command checks the specified AWS region for any issues. The command passes if there are no issues found.
+  If there are issues in the region, the command will continute to check for issues every 10 seconds until
+  no issues are dectected. If the max attempts have been reached, the commad will fail.
```
Comment (FelicianoTech) [inline src/commands/check-health.yml]: command*

File: src/commands/check-health.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+parameters:
+  aws-region-to-check:
+    type: env_var_name
+    default: AWS_DEFAULT_REGION
+    description: >
+      The AWS region to be checked for issues. It defaults to AWS_REGION that's set as an environment varaible
```
Comment (FelicianoTech) [inline src/commands/check-health.yml]: variable*

**Review by FelicianoTech — CHANGES_REQUESTED** [review]
Just minor spelling corrections needed.

**Review by JalexChen — APPROVED** [review]

File: .circleci/test-deploy.yml [hunk 3]
```diff
@@ -14,11 +14,12 @@ jobs:
       - image: cimg/base:current
     steps:
       - checkout
```
Comment (KyleTryon) [inline .circleci/test-deploy.yml]:
```suggestion
```

Checkout is not needed for this

**Review by KyleTryon — CHANGES_REQUESTED** [review]

**Pull request merged**